### PR TITLE
Optional class attribute to determine if should allow any file type

### PR DESCRIPTION
**Pull request by paulorobertocruz**
i wanted to have a simple way to upload any file type

### CODE REVIEW

**Comment by paulorobertocruz** [comment]
did i do something wrong?

File: drf_extra_fields/fields.py [hunk 1]
```diff
@@ -30,6 +30,11 @@
 
 
 class Base64FieldMixin(object):
+
+    @property
+    def ALLOW_ALL_TYPES(self):
+        return False
```
Comment (alicertel) [inline drf_extra_fields/fields.py]: How about we refer to all types with `ALLOWED_TYPES = ["*"]`. Django also uses this approach. Please see  https://docs.djangoproject.com/en/2.2/ref/settings/#allowed-hosts

Then we can use `if self.ALLOWED_TYPES is not ["*"] and file_extension not in self.ALLOWED_TYPES:` 

That way there will be one way of controlling allowed types.

How that sounds?

**Comment by alicertel** [comment]
@paulorobertocruz Thank you for your contribution. I think this is a very useful feature. I am sorry that i was too late to respond to you.

File: drf_extra_fields/fields.py (hunk 1 above)
Comment (paulorobertocruz) [reply drf_extra_fields/fields.py]: Sounds good, much better than then having two attributes controlling the same thing.

**Comment by paulorobertocruz** [comment]
@alicertel no problem. I am happy that this is helpful.

**Comment by alicertel** [comment]
@paulorobertocruz Just a reminder that i am waiting for your updates to merge this.

**Comment by paulorobertocruz** [comment]
> @paulorobertocruz Just a reminder that i am waiting for your updates to merge this.

Thank you, i needed this. Updates are coming :).

**Comment by alicertel** [comment]
@paulorobertocruz This looks great. Thank you.

Could you please add documentation and test for this update?  Sorry for not mentioning this earlier.

**Comment by paulorobertocruz** [comment]
no problem, i should have done that in the first place

**Comment by alicertel** [comment]
@paulorobertocruz Thank you.

**Comment by alicertel** [comment]
Closing the Pr since there is no progress.

**Pull request closed without merging**